conf/defaults.ini

```diff
@@ -57,6 +57,17 @@ enable_gzip = false
 cert_file =
 cert_key =
 
+# TLS settings.
+# The minimum supported version of TLS. Either "tls10", "tls11" or "tls12", default is "tls10".
+tls_min_version = tls10
```

Comment:
at the very least, default min version should be 1.1 and probably 1.2 would be better.